plugins/lxd.plugin

```diff
@@ -4,6 +4,10 @@
 init_deployment_lxd()
 {
   init_deployment_dir
+  # Minimize per default, if not specified otherwise
+  if [ -z "$MINIMIZE" ]; then
+    MINIMIZE=1
```

Comment:
where is this env var used ($MINIMIZE)?

Reply:
here for example:
https://github.com/molior-dbs/molior-tools/blob/master/molior-deploy#L744